nano/node/telemetry.hpp

```diff
@@ -27,129 +32,122 @@ namespace transport
 	class channel;
 }
 
-/*
- * Holds a response from a telemetry request
- */
-class telemetry_data_response
-{
-public:
-	nano::telemetry_data telemetry_data;
-	nano::endpoint endpoint;
-	bool error{ true };
-};
-
-class telemetry_info final
-{
-public:
-	telemetry_info () = default;
-	telemetry_info (nano::endpoint const & endpoint, nano::telemetry_data const & data, std::chrono::steady_clock::time_point last_response, bool undergoing_request);
-	bool awaiting_first_response () const;
-
-	nano::endpoint endpoint;
-	nano::telemetry_data data;
-	std::chrono::steady_clock::time_point last_response;
-	bool undergoing_request{ false };
-	uint64_t round{ 0 };
-};
-
-/*
+/**
```

Comment:
If we do not document this class now that it is fresh in the memory, we likely never will.